cloud/blockstore/libs/storage/volume/volume_ut.cpp

```diff
@@ -3632,6 +3632,115 @@ Y_UNIT_TEST_SUITE(TVolumeTest)
         }
     }
 
+    Y_UNIT_TEST(ShouldHandleDescribeBlocksRequestForMultipartitionVolume)
```

Comment:
We also need large tests for that

It may be added via separate PR, though

Reply:
added this task to issue 
https://github.com/ydb-platform/nbs/issues/356#issuecomment-1997460638